client/scala/scala-armada-client/src/test/scala/ArmadaClientSuite.scala

```diff
@@ -93,17 +112,21 @@ private class SubmitMockServer(jobMap: ConcurrentHashMap[String, Job], queueMap:
   }
 
   def submitJobs(request: JobSubmitRequest): scala.concurrent.Future[JobSubmitResponse] = {
-    val q = queueMap.get(request.queue)
-    if (q == null) {
-      val msg = "could not find queue \"" + request.queue + "\""
-      return Future.failed(new StatusRuntimeException(Status.PERMISSION_DENIED.withDescription(msg)))
+    val result: Option[Queue] = queueMap.get(request.queue)
+    result match {
+      case Some(queueFound) => {
```

Comment:
Since queueFound is not used, it is more conventional to use "_" instead.

Reply:
👍🏼  - fixed in commit ffdc1e18.